backend/modules/parameters.py

```diff
@@ -40,6 +40,8 @@ def parameters_to_command(params):
         switches.append(f'-I {params["init_img"]}')
     if 'init_mask' in params and len(params['init_mask']) > 0:
         switches.append(f'-M {params["init_mask"]}')
+    if 'init_color' in params and len(params['init_color']) > 0:
```

Comment:
Aside for @Kyle0654: Here's an example of a PR that has had to make parallel changes among the web and CLI interfaces in order to introduce a new switch. I'm looking forward to you unifying the command-line argument processing in the new architecture.